README.md

```diff
@@ -32,6 +34,8 @@ module "my_workerpool" {
 }
 ```
 
+You also need to update [variables](variables.tf) for the Lambda function to set the required `SPACELIFT_API_KEY_ID`, `SPACELIFT_API_KEY_ENDPOINT` and `SPACELIFT_WORKER_POOL_ID` parameters.
```

Comment:
Users would not update the `variables.tf` file but would provide values for those variables in the different ways Terraform supports.

Reply:
Updated the instructions and users can add these parameters to the module block